src/useComputed.ts

```diff
@@ -1,7 +1,12 @@
 import * as mobx from "mobx"
+import { IComputedValueOptions } from "mobx"
 import { useMemo } from "react"
 
-export function useComputed<T>(initialValue: () => T, inputs: ReadonlyArray<unknown> = []): T {
-    const computed = useMemo(() => mobx.computed(initialValue), inputs)
+export function useComputed<T>(
+    func: () => T,
+    inputs: ReadonlyArray<any> = [],
+    options?: IComputedValueOptions<T>

```

Comment:
for example using a struct comparer comes to mind

Reply:
Ok, but still, in such rare cases having this wrapped hook is not that useful anymore. For once you would have to pass empty array there only to be able to pass some options.

Since this is really just a one-liner, I would leave to userland.

Reply:
Another option is to give two options
(func, optionsObj?, arrayOfDeps?)
(func, arrayOfDeps?)

Reply:
Nah, overloading stinks :) Please, let's keep it simple, this is unnecessary. If the need really arise, we can always add something like `useComputedWithOptions`.

Reply:
ok, reverted